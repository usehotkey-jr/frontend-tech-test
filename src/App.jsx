// @flow

import React from 'react';
import {connect} from 'react-redux';
import {AppStyled} from "./App.styled";

export function AppComponent(props) {
    return <AppStyled>{props.app}</AppStyled>;
}

export const App = connect(
    state => state
)(AppComponent);
