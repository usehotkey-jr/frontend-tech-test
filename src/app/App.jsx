import React from "react";
import "../../node_modules/normalize.css/normalize.css";
import "../theme/shim.css";
import {AppHeader, AppRootContainer} from "./App.styled";
import {TodoCreator} from "../newTodo/TodoCreator/TodoCreator";
import {TodoList} from "../todoList/TodoList/TodoList";

/**
 * Root component
 */
export function App () {
    return (
        <AppRootContainer>
            <AppHeader>Todo List</AppHeader>
            <TodoCreator />
            <TodoList />
        </AppRootContainer>
    );
}
