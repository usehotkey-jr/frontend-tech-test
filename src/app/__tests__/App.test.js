import {createTestSnapshot} from "../../_helpers/jest/createTestSnapshot.ignore";
import {App} from "../App";

describe("App", () => {
    test("should correctly render", () => {
        createTestSnapshot.shallow(App);
    });
});
