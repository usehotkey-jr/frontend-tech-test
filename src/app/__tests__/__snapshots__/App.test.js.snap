// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should correctly render 1`] = `
<styled.div>
  <styled.h2>
    Todo List
  </styled.h2>
  <Connect(TodoCreatorComponent) />
  <Connect(TodoListComponent) />
</styled.div>
`;
